README.md

```diff
@@ -187,6 +188,76 @@ initial_qa.to_parquet('./qa.parquet', './corpus.parquet')
 
 ![rag_opt_gif](https://github.com/user-attachments/assets/55bd09cd-8420-4f6d-bc7d-0a66af288317)
 
+## 🐳 AutoRAG Docker Guide
+
+This guide provides a quick overview of building and running the AutoRAG Docker container for production, with instructions on setting up the environment for evaluation using your configuration and data paths.
+
+### 🚀 Building the Docker Image
+
+Build and run the container using `docker-compose`:
```

Comment:
No description about docker-compose here. How about just delete here, or write some description about docker-compose to running validation of dataset?

Reply:
This docker-compose file is intended only for open-source contributors. It simplifies the process of customizing AutoRAG by building a Docker image. For now, let’s temporarily remove it and include it later in the contributor documentation page.

Reply:
Okay!